.github/labeler.yml

```diff
@@ -1,5 +1,7 @@
 agent:
   - pages/agent/**/*
+cli:
+  - pages/cli/**/*
```

Comment:
I figured adding a corresponding label would be a good idea. But I don't know why its not adding it. Perhaps because the label doesn't exist yet?